src/main/java/org/zaproxy/zap/extension/jwt/utils/JWTUtils.java

```diff
@@ -291,4 +299,38 @@ public static boolean isValidJson(String value) {
         }
         return true;
     }
+
+    /**
+     * Generic utility to read contents from the file and returning the provided content as list of
+     * Strings.
+     *
+     * @param fileName
+     * @return content from file as list of strings
+     */
+    public static Set<String> readFileContentsFromResources(String fileName) {
+        Set<String> values = new HashSet<>();
+        BufferedReader bufferedReader = null;
+        try {
+            bufferedReader =
+                    new BufferedReader(
+                            new InputStreamReader(JWTUtils.class.getResourceAsStream(fileName)));
+            String inputLine;
+            while ((inputLine = bufferedReader.readLine()) != null) {
+                if (StringUtils.isNotBlank(inputLine)) {
+                    values.add(inputLine);
+                }
+            }
+        } catch (Exception ex) {
+            LOGGER.info("Unable to read publicly known secrets from: " + fileName, ex);
```

Comment:
Warn would be more noticeable than info.

Reply:
hmm sure.